projects/ngrx.io/content/guide/signals/signal-store/index.md

```diff
@@ -49,6 +49,27 @@ The `BookSearchStore` instance will contain the following properties:
 
 <div class="alert is-helpful">
 
+Similar to `signalState`, `withState` accepts user-defined `WritableSignal` 
+instances as part of the state.
```

Comment:
Let's add a bit more details here (assuming that reader is not familiar with `signalState`):

```suggestion
In addition to plain values, the `withState` feature also accepts user-defined `WritableSignal` instances as part of the state.
In this case, the SignalStore's state and the original signal remain synchronized - updating one will immediately reflect in the other.
```